modules/howtos/examples/collection-manager.js

```diff
@@ -45,21 +45,20 @@ async function go() {
   }
   // end::create-scope[]
 
-  // Note: some parts of following examples are pending https://issues.couchbase.com/browse/JSCBC-858
-
   // tag::create-collection[]
   try {
-    await collectionMgr.createCollection('example-collection', 'example-scope')
+    var collectionSpec = new couchbase.CollectionSpec({
+      name: 'example-collection',
+      scopeName: 'example-scope',
+    })
+
+    await collectionMgr.createCollection(collectionSpec)
   } catch (e) {
     if (e instanceof couchbase.CollectionExistsError) {
       console.log('The collection already exists')
-    }
-    // tag::pending-JSCBC-858[]
-    // else if (e instanceof couchbase.ScopeNotFoundError) {
-    //   console.log("The scope does not exist");
-    // }
-    // end::pending-JSCBC-858[]
-    else {
+    } else if (e instanceof couchbase.ScopeNotFoundError) {
```

Comment:
JSCBC-858 has been resolved by https://issues.couchbase.com/browse/JSCBC-905, so I've uncommented the code.

Reply:
Woohoo!